public/apps/configuration/app-router.tsx

```diff
@@ -88,22 +84,27 @@ const ROUTE_MAP: { [key: string]: RouteItem } = {
   },
 };
 
-const ROUTE_LIST = [
-  ROUTE_MAP.getStarted,
-  ROUTE_MAP[ResourceType.auth],
-  ROUTE_MAP[ResourceType.roles],
-  ROUTE_MAP[ResourceType.users],
-  ROUTE_MAP[ResourceType.serviceAccounts],
-  ROUTE_MAP[ResourceType.permissions],
-  ROUTE_MAP[ResourceType.tenants],
-  ROUTE_MAP[ResourceType.auditLogging],
-  ROUTE_MAP[ResourceType.tenantsConfigureTab],
-];
+const getRouteList = (multitenancyEnabled: boolean) => {
+  return [
+    ROUTE_MAP.getStarted,
+    ROUTE_MAP[ResourceType.auth],
+    ROUTE_MAP[ResourceType.roles],
+    ROUTE_MAP[ResourceType.users],
+    ROUTE_MAP[ResourceType.serviceAccounts],
+    ROUTE_MAP[ResourceType.permissions],
+    ...(multitenancyEnabled ? [ROUTE_MAP[ResourceType.tenants]] : []),
```

Comment:
will this add an empty entry in the list?

Reply:
It will unpack an empty list (i.e. add nothing into the list)